src/evm-canister-client/src/client.rs

```diff
@@ -209,7 +209,7 @@ impl<C: CanisterClient> EvmCanisterClient<C> {
         &self,
         address: H160,
         amount: U256,
-    ) -> CanisterClientResult<EvmResult<U256>> {
```

Comment:
@itsyaasir this is a not backward-compatible change in the API that could break the frontend. Please check it with Simeon Nortey before merging it

Reply:
Yes, I am having a talk with him